.travis.yml

```diff
@@ -39,6 +39,11 @@ after_success:
   - wget https://raw.githubusercontent.com/DiscordHooks/travis-ci-discord-webhook/master/send.sh
   - chmod +x send.sh
   - ./send.sh success $WEBHOOK_URL
+  - if [ $TRAVIS_BRANCH == "master" ] && [ $TRAVIS_EVENT_TYPE == "push" ]; then
```

Comment:
@Lulalaby if the intention is to notify our quality review for a new deployment, please put this in a `deploy` script (see the end of the file).

Reply:
Also you could send the URL instead of the word `staging`. So people just have to click on the link.